hdr/sqlite_modern_cpp.h

```diff
@@ -173,7 +174,12 @@ namespace sqlite {
       return sqlite3_sql(_stmt.get());
     }
 
-		void used(bool state) { execution_started = state; }
+		void used(bool state) {
+			if(execution_started == true && state == true) {
+				throw sqlite_exception("Already used statement executed again! Please reset() first!",sql());
```

Comment:
Hm, i like to throw stuff and i would like to replace the whole exception throwing with a user replaceable callback later (secret objective of #96 i will explain there).
But i see the point in using assert here where it not for me calling this stuff from an embedded scripting language so this can be a runtime error for me.